solutions/observability/streams/management/retention.md

```diff
@@ -1,44 +1,80 @@
 
-The **Data retention** page is made up of the following components that can help you determine how long you want your stream to retain data:
+For more information on data retention, refer to [Data stream lifecycle](../../../../manage-data/lifecycle/data-stream.md).
 
-- **Retention period**: The minimum number of days after which the data is deleted
-- **Source**: The origin of the data retention policy.
-- **Last updated**: When data retention was last updated for the selected stream.
-- **Ingestion**: Estimated ingestion per day and month, calculated based on the total size of all data in the stream  divided by the stream's age. This is an estimate, and the actual ingestion may vary.
-- **Total doc count**: The total number of documents in the stream.
-- **Ingestion Rate**: Estimated ingestion rate per time bucket. The bucket interval is dynamic and adjusts based on the selected time range. The ingestion rate is calculated using the average document size in the stream multiplied by the number of documents in each bucket. This is an estimate, and the actual ingestion rate may vary.
-- **Policy summary**: {applies_to}`stack: preview 9.1` The amount of data ingested per phase (hot, warm, cold).
+## Required permissions
 
-## Edit the data retention [streams-update-data-retention]
-From any stream page, select **Edit data retention** to change how long your data stream retains data.
+To edit data retention in {{stack}}, you need the following data stream level privileges:
+- `manage_data_stream_lifecycle`
+- `manage_ilm`
 
-### Set a specific retention period
+## Retention configuration options [streams-update-data-retention]
+Under **Retention**, select **Edit data retention** to open the configuration options. You have the following options when setting your data retention:
+
+- [Inherit from index template or parent stream](#streams-retention-inherit-from-template): Use the data retention configuration that's set in a classic stream's index template or a wired stream's parent stream.
+- [Set a specific retention period](#streams-retention-dsl): For simplicity, you can set your stream to retain data for a specific number of days. Setting a specific or indefinite retention period stores data in the hot phase for best indexing and search performance.
+- [Follow an ILM policy](#streams-retention-ilm): {applies_to}`stack: preview 9.1, ga 9.2` Select an existing ILM policy that uses phases for your data (hot, warm, cold) to allow more control when managing storage, performance, and cost as your data ages.
+
+### Inherit from index template or parent stream [streams-retention-inherit-from-template]
+If you enable **Inherit from index template** or **parent stream**, the stream uses the retention settings from its index template (for classic streams) or parent stream (for wired streams). When this option is enabled, you don’t need to specify a custom retention period or policy.
```

Comment:
Didn't check the UI for this change, but still flagging in case it needs to be corrected.

```suggestion
If you enable **Inherit from index template** or **Parent stream**, the stream uses the retention settings from its index template (for classic streams) or parent stream (for wired streams). When this option is enabled, you don’t need to specify a custom retention period or policy.
```